pkg/runtime/wfengine/activity.go

```diff
@@ -78,6 +78,7 @@ func (a *activityActor) SetActorRuntime(actorsRuntime actors.Actors) {
 // returns immediately after creating the reminder, enabling the workflow to continue processing other events
 // in parallel.
 func (a *activityActor) InvokeMethod(ctx context.Context, actorID string, methodName string, data []byte) (any, error) {
+	wfLogger.Debugf("Activity actor '%s': invoking method '%s'", actorID, methodName)
```

Comment:
all `'%s' can be changed to `%q`

Reply:
Let's keep `'%s'` for consistency... Also, it's not exactly the same: `%q` encodes some characters such as `\`, making the logs a bit harder to read